src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -987,6 +988,15 @@ void goto_instrument_parse_optionst::instrument_goto_program()
     nondet_static(ns, goto_functions);
   }
 
+  if(cmdline.isset("slice-global-inits"))
+  {
+    status() << "Slicing away initializations of unused global variables"
+      << eom;
```

Comment:
Nit picking: indent << as far as the  << in the preceding line for consistency with the rest of the code base.